docs/server.js

```diff
@@ -1,19 +1,27 @@
 const webpack = require('webpack');
 const WebpackDevServer = require('webpack-dev-server');
-const config = require('./webpack.dev.config');
+const createConfigWithPort = require('./webpack.dev.config');
 const opn = require('opn');
-const port = 3000;
+const portfinder = require('portfinder');
```

Comment:
I think you missed saving this to `package.json`

Reply:
Good spot!